apis/python/vcpkg_triplets/arm64-linux-release.cmake

```diff
@@ -0,0 +1,5 @@
+set(VCPKG_TARGET_ARCHITECTURE arm64)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE dynamic)
```

Comment:
Is there a reason for dynamic library linkage? We should better use static, per my other comments.